### PR TITLE
add peer tests

### PR DESCRIPTION
**Pull request by bogdan34**

### CODE REVIEW

File: test/lab/runner/preparer/config.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { IConstants } from 'shared/config/types';
+
+const NEVER = 1005000000;
```
Comment (axelsheva) [inline test/lab/runner/preparer/config.ts]: bad name

File: test/lab/test/peer/init.test.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import PeerController from 'core/controller/peer';
+import PeerService from 'core/service/peer';
+import { TEST_RUNNER_NAME, TEST_ASYNC_TIMEOUT } from 'test/lab/utils/constants';
+
+const TEST_NAME = 'TEST_PEER_CONNECT';
+const TEST_DONE_STAGE = 'TEST_PEER_CONNECTION_INIT_DONE';
+const TEST_AFTER_STAGE_NAME = 'TEST_PEER_CONNECTION_AFTER';
+
+describe('PEER CONNECT', function () {
+    this.timeout(DEFAULT_TEST_TIMEOUT);
```
Comment (axelsheva) [inline test/lab/test/peer/init.test.ts]: Is it used?

File: test/lab/runner/preparer/peerPreparator.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import config from 'shared/config';
+
+class PeerPreparator {
+
+    setCustomConstants = (customConstants) => {
```
Comment (macaqueee) [inline test/lab/runner/preparer/peerPreparator.ts]: add `type`

File: test/lab/runner/preparer/peerPreparator.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    };
+
+    setBlocks() {
+        console.log(`[Preparator][setBlocks]`);
+    }
+}
+
+export const preparePeerNode = (data: { customConfig?, trustedPeers? }) => {
+    return async () => {
+        const preparetor = new PeerPreparator();
```
Comment (macaqueee) [inline test/lab/runner/preparer/peerPreparator.ts]: rename to `preparator`

File: test/lab/test/peer/discover.test.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+import { TestRunner } from 'test/lab/runner';
+import { asyncTimeout } from 'shared/util/timer';
+import SystemRepository from 'core/repository/system';
+import { CUSTOM_CONFIG, PEER, } from 'test/lab/runner/preparer/config';
+import { preparePeerNode } from 'test/lab/runner/preparer/peerPreparator';
+import PeerMemoryRepository from 'core/repository/peer/peerMemory';
+import PeerController from 'core/controller/peer';
+import PeerService from 'core/service/peer';
+import { TEST_RUNNER_NAME, TEST_ASYNC_TIMEOUT } from 'test/lab/utils/constants';
+const TEST_NAME = 'TEST_PEER_DISCOVER';
```
Comment (macaqueee) [inline test/lab/test/peer/discover.test.ts]: add empty row here

File: test/lab/test/peer/pingPong.test.ts [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+import { TEST_RUNNER_NAME, TEST_ASYNC_TIMEOUT } from 'test/lab/utils/constants';
+import PeerNetworkRepository from 'core/repository/peer/peerNetwork';
+import { ActionTypes } from 'core/util/actionTypes';
+import { getRandom } from 'core/util/common';
+
+const TEST_NAME = 'TEST_PEER_PING';
+const TEST_DONE_STAGE = 'TEST_PEER_PING_DONE';
+const TEST_AFTER_STAGE_NAME = 'TEST_PEER_PING_AFTER';
+
+describe('PEER DISCOVER', function () {
```
Comment (macaqueee) [inline test/lab/test/peer/pingPong.test.ts]: rename `PEER DISCOVER` to current test name

**Pull request merged**